### ISSUE TITLE
Add search function on map

### ISSUE THREAD

**Issue by zachzwy** [trynmaps/metrics-mvp#404]
It might take some time for user to find a location on the map by just panning and zooming. Imaging a scenario that a user wants to check out and compare the transportation condition for several location for renting purpose, a searching location function would be very helpful.

**Comment by hathix** [comment]
Like searching for a street address (like in Google Maps)? That'd be a cool idea. Wonder where we would put the search bar.

**Comment by zachzwy** [comment]
Exactly. How about putting the search bar on top area of the map. And we can also add some explanation inside the text box on the top-right corner.

<img width="715" alt="Screen Shot 2019-11-08 at 8 35 32 PM" src="https://user-images.githubusercontent.com/22914663/68523033-eb40c900-0267-11ea-9b29-ea41e2eac5b4.png">

**Comment by hathix** [comment]
I like that idea of adding a search bar, Wenyu. We may even need to add two for the to/from stops.
